README.md

```diff
@@ -70,12 +83,4 @@ If your workspace contains a package that has dependencies, this extension will
 
 Additionally, the extension will monitor **Package.swift** and **Package.resolved** for changes, resolve any changes to the dependencies, and update the view as needed.
 
-> **Note**: When browsing the files in a package, Visual Studio Code may also open these files in the Explorer. If this is undesirable, open **Preferences ▸ Settings** and set **Explorer: Auto Reveal** to `false`.
-
-## Configuration
```

Comment:
You can remove this section. The configuration options will be listed in the `Feature Contributions` section and the details will be extracted from the `Package.json`